src/Jeffijoe.MessageFormat/Jeffijoe.MessageFormat.csproj

```diff
@@ -10,11 +10,11 @@
   	<RepositoryUrl>https://github.com/jeffijoe/messageformat.net</RepositoryUrl>
   	<LangVersion>latest</LangVersion>
   	<Nullable>enable</Nullable>
-  	<TargetFramework>netstandard1.1</TargetFramework>
+  	<TargetFramework>netstandard2.0</TargetFramework>
   </PropertyGroup>
 
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|AnyCPU'">
-    <DocumentationFile>bin/Release/netstandard1.1/Jeffijoe.MessageFormat.xml</DocumentationFile>
+    <DocumentationFile>bin/Release/netstandard2.0/Jeffijoe.MessageFormat.xml</DocumentationFile>
```

Comment:
This can be parameterized, which might make maintenance simpler:

```xml
<DocumentationFile>bin/$(Configuration)/$(TargetFramework)/$(AssemblyName).xml</DocumentationFile>
```

Reply:
Thanks, used this